multisig/proposals/arbitrumsepolia/vip-001/index.ts

```diff
@@ -0,0 +1,181 @@
+    },
+    {
+      target: arbitrumsepolia.RESILIENT_ORACLE,
+      signature: "acceptOwnership()",
+      params: [],
+    },
+    {
+      target: arbitrumsepolia.CHAINLINK_ORACLE,
+      signature: "acceptOwnership()",
+      params: [],
+    },
+    {
+      target: BOUND_VALIDATOR,
+      signature: "acceptOwnership()",
+      params: [],
+    },
+    {
+      target: arbitrumsepolia.CHAINLINK_ORACLE,
+      signature: "setTokenConfig((address,address,uint256))",
+      params: [[MOCK_WBTC, CHAINLINK_BTC_FEED, 144000]],
+    },
+    {
+      target: arbitrumsepolia.RESILIENT_ORACLE,
+      signature: "setTokenConfig((address,address[3],bool[3]))",
+      params: [
+        [
+          MOCK_WBTC,
+          [
+            arbitrumsepolia.CHAINLINK_ORACLE,
+            "0x0000000000000000000000000000000000000000",
+            "0x0000000000000000000000000000000000000000",
+          ],
+          [true, false, false],
+        ],
+      ],
+    },
+    {
+      target: arbitrumsepolia.CHAINLINK_ORACLE,
+      signature: "setTokenConfig((address,address,uint256))",
+      params: [[MOCK_WETH, CHAINLINK_ETH_FEED, 144000]],
+    },
+    {
+      target: arbitrumsepolia.RESILIENT_ORACLE,
+      signature: "setTokenConfig((address,address[3],bool[3]))",
+      params: [
+        [
+          MOCK_WETH,
+          [
+            arbitrumsepolia.CHAINLINK_ORACLE,
+            "0x0000000000000000000000000000000000000000",
+            "0x0000000000000000000000000000000000000000",
+          ],
+          [true, false, false],
+        ],
+      ],
+    },
+    {
+      target: arbitrumsepolia.CHAINLINK_ORACLE,
+      signature: "setTokenConfig((address,address,uint256))",
+      params: [[MOCK_USDC, CHAINLINK_USDC_FEED, 144000]],
+    },
+    {
+      target: arbitrumsepolia.RESILIENT_ORACLE,
+      signature: "setTokenConfig((address,address[3],bool[3]))",
+      params: [
+        [
+          MOCK_USDC,
+          [
+            arbitrumsepolia.CHAINLINK_ORACLE,
+            "0x0000000000000000000000000000000000000000",
+            "0x0000000000000000000000000000000000000000",
+          ],
+          [true, false, false],
+        ],
+      ],
+    },
+    {
+      target: arbitrumsepolia.CHAINLINK_ORACLE,
+      signature: "setTokenConfig((address,address,uint256))",
+      params: [[MOCK_USDT, CHAINLINK_USDT_FEED, 144000]],
+    },
+    {
+      target: arbitrumsepolia.RESILIENT_ORACLE,
+      signature: "setTokenConfig((address,address[3],bool[3]))",
+      params: [
+        [
+          MOCK_USDT,
+          [
+            arbitrumsepolia.CHAINLINK_ORACLE,
+            "0x0000000000000000000000000000000000000000",
+            "0x0000000000000000000000000000000000000000",
+          ],
+          [true, false, false],
+        ],
+      ],
+    },
+    {
+      target: arbitrumsepolia.CHAINLINK_ORACLE,
+      signature: "setTokenConfig((address,address,uint256))",
+      params: [[MOCK_ARB, CHAINLINK_ARB_FEED, 144000]],
```

Comment:
I would set 4200 seconds for the stale period here (heartbeat of 1 hour)

Reply:
What if we set it at `6000`seconds(`100`minutes) as we have set [here](https://github.com/VenusProtocol/vips/blob/b8325d5d0a61136f12cac100b4c8540f1d1b6c93/multisig/proposals/ethereum/vip-001/index.ts#L10)?

Reply:
Done https://github.com/VenusProtocol/vips/pull/247/commits/c573c14b8980fedd4ab11d46d064198f238595b3